go.mod

```diff
@@ -42,7 +42,7 @@ require (
 	k8s.io/klog/v2 v2.30.0
 	k8s.io/kubectl v0.23.0
 	k8s.io/utils v0.0.0-20210930125809-cb0fa318a74b
-	sigs.k8s.io/cluster-api v1.1.0
+	sigs.k8s.io/cluster-api v1.1.1-0.20220203212859-13fcc674c7ab
```

Comment:
@alvaroaleman v1.1.1 is out with this change, we should be able to update and merge this now

thanks again

Reply:
Thanks, updated